t/plugin/zipkin.t

```diff
@@ -0,0 +1,299 @@
+use t::APISix 'no_plan';
+
+repeat_each(2);
+no_long_string();
+no_root_location();
+log_level("info");
+run_tests;
+
+__DATA__
+
+=== TEST 1: sanity
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.zipkin")
+            local ok, err = plugin.check_schema({endpoint = 'http://127.0.0.1', sample_ratio = 0.001})
+            if not ok then
+                ngx.say(err)
+            end
+
+            ngx.say("done")
+        }
+    }
+--- request
+GET /t
+--- response_body
+done
+--- no_error_log
+[error]
+
+
+
+=== TEST 2: default value of sample_ratio
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.zipkin")
+            local ok, err = plugin.check_schema({endpoint = 'http://127.0.0.1'})
+            if not ok then
+                ngx.say(err)
+            end
+
+            ngx.say("done")
+        }
+    }
+--- request
+GET /t
+--- response_body
+done
+--- no_error_log
+[error]
+
+
+
+=== TEST 3: wrong value of ratio
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.zipkin")
+            local ok, err = plugin.check_schema({endpoint = 'http://127.0.0.1', sample_ratio = -0.1})
+            if not ok then
+                ngx.say(err)
+            end
+
+            ngx.say("done")
+        }
+    }
+--- request
+GET /t
+--- response_body
+invalid "minimum" in docuement at pointer "#/sample_ratio"
+done
+--- no_error_log
+[error]
+
+
+
+=== TEST 4: wrong value of ratio
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.zipkin")
+            local ok, err = plugin.check_schema({endpoint = 'http://127.0.0.1', sample_ratio = 2})
+            if not ok then
+                ngx.say(err)
+            end
+
+            ngx.say("done")
+        }
+    }
+--- request
+GET /t
+--- response_body
+invalid "maximum" in docuement at pointer "#/sample_ratio"
+done
+--- no_error_log
+[error]
+
+
+
+=== TEST 5: add plugin
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
```

Comment:
the size of code block is more than 1k, it may failed. we need to make it smaller.

Reply:
need to fix other similar issues.